js/users.js

```diff
@@ -377,12 +377,16 @@ var UserList = {
 		$userListBody.on('click', '.delete', function () {
 			// Call function for handling delete/undo
 			var uid = UserList.getUID(this);
-			UserDeleteHandler.mark(uid);
-		});
-
-		//delete a marked user when leaving the page
-		$(window).on('beforeunload', function () {
-			UserDeleteHandler.deleteEntry();
+			OC.dialogs.confirm(
+				t('user_management', 'You are about to delete a user. This action can\'t be undone and is permanent. All user data, files and shares will be deleted. Are you sure that you want to permanently delete {userName}?', {userName: uid}),
```

Comment:
Add space between username and question mark.

As far as I remember in English we always have a space there.

Reply:
```
Are you sure that you want to permanently delete user123?
```
is fine.

There is no space before a ``?``

Reply:
@PVince81 @phil-davis 

For instance lets take a look at https://www.gnu.org/philosophy/free-sw.html. 

- The heading `What is free software?` does not have space before `?`. 
- Another instance `Have a question about free software licensing not answered here?` in the same page.

Let me know if we agree on no space before `?` :)

Reply:
@PVince81 already clicked the approve button. He will be happy! I will mark this discussion resolved.